napari/_app_model/_app.py

```diff
@@ -57,5 +59,19 @@ def _public_types(module):
 
 
 def get_app() -> NapariApplication:
+    """Get the Napari Application singleton."""
```

Comment:
As with the other, lets just say deprecated, use `get_app_model`, to deprecate 0.6.0